recipes/networkit/meta.yaml

```
@@ -0,0 +1,70 @@
+{% set name = "networkit" %}
+{% set version = "5.0" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "aefef17589991f6e4a04451164eb2d379c7301c9c85e787ef3ddf2f4777b922d" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  - git_url: 'https://github.com/kit-parco/{{name}}.git'
+    git_tag: '{{version}}'
+  
+build:
+  skip: True  # [py27 or py33 or win or win_64]
+  number: 0
+  script:
+    - '{{GIT}} submodules update --init'
+    - '{{PYTHON}} -m pip install --ignore-installed -vv .'
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake >=3.5
+    - ninja >=1.9.0
+  host:
+    - python >=3.3
```

The Python version should only be constrained in this way when the package is `noarch`
```suggestion
    - python
```
(If not you have one build per Python version)